### DISCUSSION TITLE
Umbraco identity login always goes to umbraco#/content

### DISCUSSION THREAD

**Discussion by james-whittington1** [umbraco/Umbraco-CMS#15948] [Features and ideas]
We are using publish notifications and some of our editors have complained about the login process that takes place when click the link in the notification email. 

When they click the link, they are prompted to login, and the redirected to the root of the content section, instead of the page that is indicated in the initial link.

During the login process i can see that the correct location is included in the url sent to the identity provider, but it does look like it could be malformed. 

I would expect this to work, and have tested it on several different Umbraco 13 cloud instances. Is there something I need to configure to get this working? 

I have include a redacted Umbraco identity url that the back office loads when logging in just in case it is useful
https://identity.umbraco.com/<snip>/b2c_1a_signinbackoffice/oauth2/v2.0/authorize?client_id=<snip>&redirect_uri=<spin>&response_type=<snip>&response_mode=form_post&nonce=<snip>&client_info=1&x-client-brkrver=<snip>&environment_id=<snip>&project_name=<snip>.umbraco.io&state=<snip>&x-client-SKU=ID_NET8_0&**x-client-ver=7.0.3.0#/content/content/edit/1375**

I am unsure where to actually create an issue, I am thinking it could be to do with any (or a combination of) Umbraco CMS, Umbraco Cloud, or Umbraco Identity. 

Any help or pointers appreciated.

**Comment by jdpnielsen** [comment]
I will second this - we need support for direct linking when integrating with certain 3rd party systems.
